README.md

```diff
@@ -4,19 +4,29 @@
 
 ## Overview
 
-This project involves creating a responsive website for Pet Care company that features a modern design with a navigation menu, company logo, title header, and a central GIF. The website is built using HTML, Tailwind css and JavaScript
+This project involves creating a responsive website for Pet Care company that features a modern design with a navigation menu, company logo, title header, and a central GIF. The website is built using HTML, Tailwind css and JavaScript using Jest as testing enviroment.
 
 ## Setup and Usage
 
 1. Clone the repository to your local machine.
 2. Run `docker-compose up -d` OR `docker compose up -d`.
-3. Visit `http://localhost:8081/home.html`
+3. Visit `http://localhost:8081/home.html`.
 4. You are all set up to edit the webpage! Enjoy 😄
 
+## Setup testing enviroment (optional)
+1. Run `npm init -y` to initialize default value.
```

Comment:
Any other developer doesn't need to do this. What does this command do? It creates a package.json file correct? This file is now in git. 
If i run this command I would probably get an error.